eng/Versions.props

```diff
@@ -80,6 +80,7 @@
     <SystemPrivateRuntimeUnicodeDataVersion>5.0.0-beta.20319.2</SystemPrivateRuntimeUnicodeDataVersion>
     <SystemSecurityCryptographyX509CertificatesTestDataVersion>5.0.0-beta.20319.2</SystemSecurityCryptographyX509CertificatesTestDataVersion>
     <SystemWindowsExtensionsTestDataVersion>5.0.0-beta.20319.2</SystemWindowsExtensionsTestDataVersion>
+    <SystemRuntimeTestDataVersion>5.0.0-beta.20319.2</SystemRuntimeTestDataVersion>
```

Comment:
I don't think we need this anymore, right?

Reply:
yes I can remove that